tests/core/test_settings.py

```diff
@@ -228,9 +228,11 @@ def test_uses_updated_session_timeout_config_by_default(self):
         ("sqlite://", contextlib.nullcontext()),
     ],
 )
-def test_sqlite_relative_path(monkeypatch, value, expectation):
+def test_sqlite_relative_path(value, expectation):
```

Comment:
This monkeypatch was pretty bad when this test runs with others as it had a lot of side-effects. Settings _AIRFLOW_SKIP_DB_TEST and patching with context for what's modified in configure_orm made it side-effect free.